pyccl/nl_pt/bacco_lbias.py

```diff
@@ -181,6 +191,12 @@ def update_ingredients(self, cosmo):
         # save templates in a table
         self.lpt_table = lpt_table
 
+        # Get linear power spectrum from baccoemu
```

Comment:
as before, should this be encased in `if extrap_lin`? If I follow the code logic, if `extrap_lin` was `False` you'd still calculate this but then multiply it by zero, right? I guess this is fine if all emulator evaluations are super fast. Just want to make sure this is what you intended! (in which case, fine not to add the if checks)